safehttp/mux_test.go

```diff
@@ -324,16 +400,23 @@ func (setHeaderConfig) Match(i safehttp.Interceptor) bool {
 type setHeaderConfigInterceptor struct{}
 
 func (p setHeaderConfigInterceptor) Before(w *safehttp.ResponseWriter, _ *safehttp.IncomingRequest, cfg interface{}) safehttp.Result {
-	name := "Pizza"
 	value := "Hawaii"
 	if c, ok := cfg.(setHeaderConfig); ok {
-		name = c.name
-		value = c.value
+		value = c.pizzaValue
 	}
-	w.Header().Set(name, value)
+	w.Header().Set("Pizza", value)
 	return safehttp.Result{}
 }
 
+func (p setHeaderConfigInterceptor) Commit(w *safehttp.CommitResponseWriter, r *safehttp.IncomingRequest, resp safehttp.Response, cfg interface{}) safehttp.Result {
+	value := "Fusili"
+	if c, ok := cfg.(setHeaderConfig); ok {
+		value = c.pastaValue
+	}
```

Comment:
I would honestly use a type assertion without check. If this assertion fails we have a big bug in the framework and I'd rather have this panic.

Reply:
Ah, yes, this check was to check if a valid config was passed. If there was no valid config passed then the default values would be used.